ui-v2/src/components/ui/block.tsx

```diff
@@ -0,0 +1,20 @@
+import { cn } from "@/lib/utils";
+import { createElement, forwardRef } from "react";
+import { UtilityProps, spacingUtiltiesClasses } from "./utils/spacing-utils";
+
+type Props = UtilityProps & {
```

Comment:
Should we omit some of the `UtilityProps` for `Block`? I'm thinking that stuff like `flex-direction` and `align-items` don't apply when display is `block`. Also, maybe we should emit the `display` prop so this can only be `block` display.

Reply:
ya, I think thats a good idea

Reply:
updated in latest push